### ISSUE TITLE
[Bug]: Qwen3.6-27B 启动bf16权重推理,第一次推理无法返回结果

### ISSUE THREAD

**Issue by darrensun06** [vllm-project/vllm-ascend#9355]
### Your current environment

<details>
<summary>

</summary>

```text
Collecting environment information...
PyTorch version: 2.9.0+cpu
Is debug build: False

OS: Ubuntu 22.04.5 LTS (aarch64)
GCC version: (Ubuntu 11.4.0-1ubuntu1~22.04.3) 11.4.0
Clang version: 15.0.7
CMake version: version 4.3.2
Libc version: glibc-2.35

Python version: 3.11.14 (main, Feb 26 2026, 04:49:14) [GCC 11.4.0] (64-bit runtime)
Python platform: Linux-4.19.90-2102.2.0.0066.ctl2.aarch64-aarch64-with-glibc2.35

CPU:
Architecture:                    aarch64
CPU op-mode(s):                  64-bit
Byte Order:                      Little Endian
CPU(s):                          192
On-line CPU(s) list:             0-191
Vendor ID:                       HiSilicon
Model name:                      Kunpeng-920
Model:                           0
Thread(s) per core:              1
Core(s) per cluster:             48
Socket(s):                       -
Cluster(s):                      4
Stepping:                        0x1
CPU max MHz:                     2600.0000
CPU min MHz:                     200.0000
BogoMIPS:                        200.00
Flags:                           fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma dcpop asimddp asimdfhm ssbs
L1d cache:                       12 MiB (192 instances)
L1i cache:                       12 MiB (192 instances)
L2 cache:                        96 MiB (192 instances)
L3 cache:                        192 MiB (8 instances)
NUMA node(s):                    8
NUMA node0 CPU(s):               0-23
NUMA node1 CPU(s):               24-47
NUMA node2 CPU(s):               48-71
NUMA node3 CPU(s):               72-95
NUMA node4 CPU(s):               96-119
NUMA node5 CPU(s):               120-143
NUMA node6 CPU(s):               144-167
NUMA node7 CPU(s):               168-191
Vulnerability Itlb multihit:     Not affected
Vulnerability L1tf:              Not affected
Vulnerability Mds:               Not affected
Vulnerability Meltdown:          Not affected
Vulnerability Spec store bypass: Mitigation; Speculative Store Bypass disabled via prctl
Vulnerability Spectre v1:        Mitigation; __user pointer sanitization
Vulnerability Spectre v2:        Not affected
Vulnerability Srbds:             Not affected
Vulnerability Tsx async abort:   Not affected

Versions of relevant libraries:
[pip3] numpy==1.26.4
[pip3] pyzmq==27.1.0
[pip3] torch==2.9.0+cpu
[pip3] torch_npu==2.9.0.post1+gitee7ba04
[pip3] torchaudio==2.9.0
[pip3] torchvision==0.24.0
[pip3] transformers==4.57.6
[pip3] triton-ascend==3.2.0.dev20260322
[conda] Could not collect
vLLM Version: 0.18.0
vLLM Ascend Version: 0.18.0

ENV Variables:
ASCEND_TOOLKIT_LATEST_HOME=/usr/local/Ascend/ascend-toolkit/latest
ATB_OPSRUNNER_KERNEL_CACHE_LOCAL_COUNT=1
ATB_STREAM_SYNC_EVERY_RUNNER_ENABLE=0
ATB_STREAM_SYNC_EVERY_KERNEL_ENABLE=0
ATB_OPSRUNNER_KERNEL_CACHE_GLOABL_COUNT=5
VLLM_USE_MODELSCOPE=true
PYTORCH_NPU_ALLOC_CONF=expandable_segments:True
ATB_HOME_PATH=/usr/local/Ascend/nnal/atb/latest/atb/cxx_abi_1
ASCEND_TOOLKIT_HOME=/usr/local/Ascend/cann-8.5.1
ATB_COMPARE_TILING_EVERY_KERNEL=0
ASCEND_OPP_PATH=/usr/local/Ascend/cann-8.5.1/opp
LD_LIBRARY_PATH=/usr/local/Ascend/nnal/atb/latest/atb/cxx_abi_1/lib:/usr/local/Ascend/nnal/atb/latest/atb/cxx_abi_1/examples:/usr/local/Ascend/nnal/atb/latest/atb/cxx_abi_1/tests/atbopstest:/usr/local/Ascend/ascend-toolkit/latest/tools/aml/lib64:/usr/local/Ascend/ascend-toolkit/latest/tools/aml/lib64/plugin:/usr/local/Ascend/ascend-toolkit/latest/lib64:/usr/local/Ascend/ascend-toolkit/latest/lib64/plugin/opskernel:/usr/local/Ascend/ascend-toolkit/latest/lib64/plugin/nnengine:/usr/local/Ascend/ascend-toolkit/latest/opp/built-in/op_impl/ai_core/tbe/op_tiling:/usr/local/Ascend/cann-8.5.1/tools/aml/lib64:/usr/local/Ascend/cann-8.5.1/tools/aml/lib64/plugin:/usr/local/Ascend/cann-8.5.1/lib64:/usr/local/Ascend/cann-8.5.1/lib64/plugin/opskernel:/usr/local/Ascend/cann-8.5.1/lib64/plugin/nnengine:/usr/local/Ascend/cann-8.5.1/opp/built-in/op_impl/ai_core/tbe/op_tiling:/usr/local/Ascend/driver/lib64:/usr/local/Ascend/driver/lib64/common/:/usr/local/Ascend/driver/lib64/driver/:/usr/local/python3.11.14/lib::/usr/local/lib
ASCEND_AICPU_PATH=/usr/local/Ascend/cann-8.5.1
ATB_STREAM_SYNC_EVERY_OPERATION_ENABLE=0
ASCEND_HOME_PATH=/usr/local/Ascend/cann-8.5.1
ATB_MATMUL_SHUFFLE_K_ENABLE=1
ATB_WORKSPACE_MEM_ALLOC_ALG_TYPE=1
ATB_SHARE_MEMORY_NAME_SUFFIX=
TORCH_DEVICE_BACKEND_AUTOLOAD=1
PYTORCH_NVML_BASED_CUDA_CHECK=1
TORCHINDUCTOR_COMPILE_THREADS=1


NPU:
+------------------------------------------------------------------------------------------------+
| npu-smi 25.5.2                   Version: 25.5.2                                               |
+---------------------------+---------------+----------------------------------------------------+
| NPU   Name                | Health        | Power(W)    Temp(C)           Hugepages-Usage(page)|
| Chip                      | Bus-Id        | AICore(%)   Memory-Usage(MB)  HBM-Usage(MB)        |
+===========================+===============+====================================================+
| 2     910B3               | OK            | 89.8        52                0    / 0             |
| 0                         | 0000:81:00.0  | 0           0    / 0          52434/ 65536         |
+===========================+===============+====================================================+
| 3     910B3               | OK            | 98.5        51                0    / 0             |
| 0                         | 0000:82:00.0  | 0           0    / 0          52433/ 65536         |
+===========================+===============+====================================================+
+---------------------------+---------------+----------------------------------------------------+
| NPU     Chip              | Process id    | Process name             | Process memory(MB)      |
+===========================+===============+====================================================+
| 2       0                 | 14798         | VLLMWorker_TP            | 49063                   |
+===========================+===============+====================================================+
| 3       0                 | 14799         | VLLMWorker_TP            | 49063                   |
+===========================+===============+====================================================+

CANN:
package_name=Ascend-cann-toolkit
version=8.5.1
innerversion=V100R001C25SPC002B220
compatible_version=[V100R001C15],[V100R001C18],[V100R001C19],[V100R001C20],[V100R001C21],[V100R001C23]
arch=aarch64
os=linux
path=/usr/local/Ascend/cann-8.5.1

```

</details>

### 🐛 Describe the bug

curl请求：
curl -X POST http://localhost:6710/v1/chat/completions \
  -H "Content-Type: application/json" \
  -d '{
    "model": "Qwen3.6-27B",
    "messages": [
      {"role": "user", "content": "你是谁？"}
    ],
    "temperature": 0.7,
    "max_tokens": 1024,
    "chat_template_kwargs": {
    "enable_thinking": false
    }
  }'

首次推理：结果中的content为空
[root@A800T-A2-1 qwen-project]# bash curl.sh
开始时间：Tue May 19 11:54:42 AM UTC 2026
{"id":"chatcmpl-aa26f49b43cda1e8","object":"chat.completion","created":1779191682,"model":"Qwen3.6-27B","choices":[{"index":0,"message":{"role":"assistant","content":"","refusal":null,"annotations":null,"audio":null,"function_call":null,"tool_calls":[],"reasoning":null},"logprobs":null,"finish_reason":"stop","stop_reason":null,"token_ids":null}],"service_tier":null,"system_fingerprint":null,"usage":{"prompt_tokens":14,"total_tokens":15,"completion_tokens":1,"prompt_tokens_details":null,"completion_tokens_details":{"reasoning_tokens":0,"audio_tokens":null,"accepted_prediction_tokens":null,"rejected_prediction_tokens":null}},"prompt_logprobs":null,"prompt_token_ids":null,"kv_transfer_params":null}
结束时间：Tue May 19 11:54:44 AM UTC 2026

之后的推理请求正常：
[root@A800T-A2-1 qwen-project]# bash curl.sh
开始时间：Tue May 19 11:54:53 AM UTC 2026
{"id":"chatcmpl-bdcfd2d18c70f125","object":"chat.completion","created":1779191693,"model":"Qwen3.6-27B","choices":[{"index":0,"message":{"role":"assistant","content":"我是通义千问（Qwen），是由阿里巴巴集团通义实验室独立研发的大语言模型。\n\n请问有什么我可以帮您的吗？","refusal":null,"annotations":null,"audio":null,"function_call":null,"tool_calls":[],"reasoning":null},"logprobs":null,"finish_reason":"stop","stop_reason":null,"token_ids":null}],"service_tier":null,"system_fingerprint":null,"usage":{"prompt_tokens":14,"total_tokens":44,"completion_tokens":30,"prompt_tokens_details":null,"completion_tokens_details":{"reasoning_tokens":0,"audio_tokens":null,"accepted_prediction_tokens":null,"rejected_prediction_tokens":null}},"prompt_logprobs":null,"prompt_token_ids":null,"kv_transfer_params":null}
结束时间：Tue May 19 11:55:09 AM UTC 2026

异常的首次推理请求对应的服务日志信息：
(APIServer pid=276) DEBUG 05-19 11:54:42 [v1/sample/logits_processor/__init__.py:63] No logitsprocs plugins installed (group vllm.logits_processors).
(EngineCore pid=299) DEBUG 05-19 11:54:42 [v1/engine/core.py:1161] EngineCore loop active.
(EngineCore pid=299) DEBUG 05-19 11:54:42 [distributed/device_communicators/shm_broadcast.py:191] Poller received notify event
(Worker_TP0 pid=312) DEBUG 05-19 11:54:42 [distributed/device_communicators/shm_broadcast.py:191] Poller received notify event
(Worker_TP1 pid=313) DEBUG 05-19 11:54:42 [distributed/device_communicators/shm_broadcast.py:191] Poller received notify event
(Worker_TP1 pid=313) DEBUG 05-19 11:54:42 [model_runner_v1.py:1213] Running batch with cudagraph_mode: NONE, batch_descriptor: BatchDescriptor(num_tokens=14, num_reqs=None, uniform=False, has_lora=False, num_active_loras=0), should_ubatch: False, num_tokens_across_dp: None
(Worker_TP0 pid=312) DEBUG 05-19 11:54:42 [model_runner_v1.py:1213] Running batch with cudagraph_mode: NONE, batch_descriptor: BatchDescriptor(num_tokens=14, num_reqs=None, uniform=False, has_lora=False, num_active_loras=0), should_ubatch: False, num_tokens_across_dp: None
(EngineCore pid=299) DEBUG 05-19 11:54:44 [distributed/device_communicators/shm_broadcast.py:191] Poller received notify event
(APIServer pid=276) INFO:     127.0.0.1:53520 - "POST /v1/chat/completions HTTP/1.1" 200 OK
(Worker_TP1 pid=313) DEBUG 05-19 11:54:44 [model_runner_v1.py:1213] Running batch with cudagraph_mode: FULL, batch_descriptor: BatchDescriptor(num_tokens=3, num_reqs=1, uniform=True, has_lora=False, num_active_loras=0), should_ubatch: False, num_tokens_across_dp: None
(Worker_TP0 pid=312) DEBUG 05-19 11:54:44 [model_runner_v1.py:1213] Running batch with cudagraph_mode: FULL, batch_descriptor: BatchDescriptor(num_tokens=3, num_reqs=1, uniform=True, has_lora=False, num_active_loras=0), should_ubatch: False, num_tokens_across_dp: None
(Worker_TP1 pid=313) INFO 05-19 11:54:44 [acl_graph.py:196] Replaying aclgraph
(Worker_TP0 pid=312) INFO 05-19 11:54:44 [acl_graph.py:196] Replaying aclgraph
(EngineCore pid=299) DEBUG 05-19 11:54:45 [distributed/device_communicators/shm_broadcast.py:191] Poller received notify event
(APIServer pid=276) INFO 05-19 11:54:47 [v1/metrics/loggers.py:259] Engine 000: Avg prompt throughput: 1.4 tokens/s, Avg generation throughput: 0.1 tokens/s, Running: 0 reqs, Waiting: 0 reqs, GPU KV cache usage: 0.0%, Prefix cache hit rate: 0.0%

服务启动参数:
vllm serve /model \
  --host 0.0.0.0 \
  --port 6710\
  --data-parallel-size 1 \
  --tensor-parallel-size 2 \
  --seed 1024 \
  --served-model-name Qwen3.6-27B \
  --max-num-seqs 64 \
  --max-model-len 262144 \
  --max-num-batched-tokens 8096 \
  --trust-remote-code \
  --reasoning-parser qwen3 \
  --tool-call-parser qwen3_coder \
  --enable-auto-tool-choice \
  --gpu-memory-utilization ${GPU_MEM_UTIL:-0.8} \
  --no-enable-prefix-caching \
  --speculative_config '{"method":"qwen3_next_mtp","num_speculative_tokens":2,"disable_padded_drafter_batch":false}' \
  --compilation-config '{"cudagraph_mode":"FULL_DECODE_ONLY","cudagraph_capture_sizes":[3,6,9,12,15,18,21,24,27,30,33,36,39,42,45,48,51,54,57,60,63,66,69,72,75,78,81,84,87,90,93,96]}' \
  --additional-config '{"enable_cpu_binding":true}' 2>&1 | tee ${LOG_PATH}/vllm_server.log

**Comment by darrensun06** [comment]
补充信息：尝试修改请求内容，修改为如下后，可以正常得到推理结果，请定位问题原因
curl -X POST http://localhost:6006/v1/chat/completions \
  -H "Content-Type: application/json" \
  -d '{
    "model": "Qwen3.6-27B",
    "messages": [
      {"role": "user", "content": "可以给我介绍一下杭州吗？"}
    ],
    "temperature": 0.9,
    "max_tokens": 1024,
    "chat_template_kwargs": {
      "enable_thinking": false
    }
  }'

**Comment by Tflowers-0129** [comment]
@LoganJane Please take a look about this issue

**Comment by runmana** [comment]
请问用的哪个镜像啊

**Comment by darrensun06** [comment]
版本信息：quay.io/ascend/vllm-ascend:v0.18.0
